frontend/src/pages/user/UserAnswerPage.js

```diff
@@ -121,8 +126,9 @@ export default function UserAnswerPage() {
             </div>
             {question.choices.map((choice, index) => (
               <div key={index} className="flex-1 grid gap-5">
-                {["A", "B", "C", "D"].map((letter) => (
+                {["A", "B", "C", "D"].map((letter, ind) => (
```

Comment:
idx or index is way more clearer
```suggestion
                {["A", "B", "C", "D"].map((letter, idx) => (
```